packages/discordx/src/Client.ts

```diff
@@ -1201,8 +1201,9 @@ export class Client extends ClientJS {
 
     const matchedPrefix = prefixRegex.exec(message.content)?.at(1) ?? "unknown";
     const isPrefixBaseCommand = mappedPrefix.includes(matchedPrefix);
-    const contentWithoutPrefix =
-      message.content.replace(prefixRegex, "").trim() + " ";
+    const contentWithoutPrefix = `${message.content
```

Comment:
why?

Reply:
this just needs to be `message.content.replace(prefixRegex, "").trim()`

Reply:
no need to wrap it in a template string

Reply:
there is a space at the end

Reply:
just noticed, sorry